### PR TITLE
route: Add route for adding to cart

### PR DESCRIPTION
**Pull request by sohailrajdev97**

### CODE REVIEW

**Comment by Nischay-Pro** [comment]
Update naming style before committing. Also do run checks :P

File: utils/service.constructor.js [hunk 1]
```diff
@@ -37,6 +38,92 @@ function service(model, router) {
...
+									eventTeams.push(team_id);
+
+									eventModel.update({_id: event_id}, {teams: eventTeams}, function(err, num){
+										if(err)
+										{
+											res.json({status: 500, msg: "Error"});
+										}
+										else
+										{
+											res.json({status: 200, msg: "Successful"});
```
Comment (aero31aero) [inline utils/service.constructor.js]: Use `res.status(200)` as well to give the correct HTTP status. Same for others. Give status 404 when resource isn't found.

**Comment by aero31aero** [comment]
LGTM. Left a few minor comments. :)

**Comment by aero31aero** [comment]
Merged. Thanks @sohailrajdev97 !

**Pull request merged**